charts/misc-apps/examples/signalilo.yaml

```
@@ -0,0 +1,24 @@
+signalilo:
+  enabled: true
+  project: infra-signalilo
+  values:
+    resources:
+      limits:
+        cpu: 50m
```

```suggestion
        cpu: 100m
```
The original value causes the following error:
`Deployment.apps "signalilo" is invalid: spec.template.spec.containers[0].resources.requests: Invalid value: "100m": must be less than or equal to cpu limit`
This makes perfect sense, since the upstream value for `resources.requests.cpu` is indeed `100m`.
So the proposed value in this example is not sensible, so I suggest to increase it. Maybe one could also set it to `200m`, which would be the upstream default limit.